content/authors/aidinzp/_index.md

```diff
@@ -0,0 +1,51 @@
+---
+# Display name
+title: Aidin Zaherparandaz
+
+# Username (this should match your github username)
+authors:
+- aidinzp
+
+# Role/position e.g. Graduate student
+role: Recent Grad
+
+# Organization membership
+Organizations/Affiliations:
+organizations:
+- name: University of Manitoba
+  url: https://www.google.ca
+
+# A list of your interests, add as many as you want
+interests:
+- Machine Learning 
+- Convolutional Neural Networks
+- Partial Differential Equations
+
+# Your educational history, add as many as you want
+education:
+  courses:
+  - course: MSc - Mathematics
+    institution: Univerisity of Manitoba
+  - course: BSc - Industrial Mathematics
+    institution: Sharif University of Technology
+
+# Social/Academic Networking, delete any items you don't want to use
+social:
+- icon: github
+  icon_pack: fab
+  link: https://github.com/aidinzp
+#- icon: twitter
+#  icon_pack: fab
+#  link: https://twitter.com/carlosshigoto
+- icon: linkedin
+  icon_pack: fab
+ link: https://www.linkedin.com/in/aidinzp/
+
+user_groups:
+- committee: members
+  role: Team 4 
+
+tags:
+- '2021'
+Interested in various fields of Mathematics and Data Science, specifically Machine Learning, Convolutional Neural Networks, as well as Partial Differential Equations. Have been working on some projects for the past few months.
```

Comment:
I think the stuff on this line should be moved to the end of the file (after the line which says `---`). The area after `---` supports markdown.